### PR TITLE
docs: add docs for ORC tables

### LINKED ISSUE #206: Create docs page on ORC tables
Create a docs page on ORC tables and explain the advantages/disadvantages of ORC.

### PR DESCRIPTION
**Pull request by avriiil**
This PR adds a documentation page that shows users how to work with ORC tables. Will close #206

### CODE REVIEW

File: docs/usage/tables/orc.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Run the command below with the correct `path/to/storage` to create a new ORC table with 2 colummns: `some_numbers` and `some_letters`.
+
+```sh
+bin/uc table create --full_name unity.default.test --columns "some_numbers INT, some_letters STRING" --storage_location $DIRECTORY$ --format ORC
+```
+
+Note that you will need to manually set the $DIRECTORY$ variable to the correct storage location. If you don't know where Unity Catalog is storing your files, then take a look at the metadata of an existing table using `bin/uc table get --full_name <catalog.schema.table>` to see its storage location.
+
+After you run the `table create` command, your output should look something like this:
```
[the remark this answers is not in the document]
Comment (avriiil) [reply docs/usage/tables/orc.md:21]: Yes, done!

**Comment by avriiil** [comment]
Thanks for the review @jaceklaskowski ! I've incorporated your suggestions.

File: docs/usage/tables/orc.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# ORC
+
+This page explains how you can work with ORC tables in your Unity Catalog.
+
+Let's take a look at how we can create ORC tables in Unity Catalog.
```
Comment (jaceklaskowski) [inline docs/usage/tables/orc.md:5]: Remove it = redundant

File: docs/usage/tables/orc.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# ORC
+
+This page explains how you can work with ORC tables in your Unity Catalog.
```
Comment (jaceklaskowski) [inline docs/usage/tables/orc.md:3]: Remove `your`

File: docs/usage/tables/orc.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Let's take a look at how we can create ORC tables in Unity Catalog.
+
+## Create ORC table
+
+Use `bin/uc table create` command with the `--format ORC` option to create a new ORC table in Unity Catalog.
+
+Use the following command to create an external ORC tables (at `path/to/storage` location) with 2 colummns: `some_numbers` and `some_letters`.
```
Comment (jaceklaskowski) [inline docs/usage/tables/orc.md:11]: Can you match this PR to the AVRO's, please? 🙏  The pages should really be identical, shouldn't they?

**Review by jaceklaskowski — CHANGES_REQUESTED** [review]

**Comment by avriiil** [comment]
@jaceklaskowski @MrPowers - this is ready for another pass. Layout now matches with Avro page.

**Review by MrPowers — APPROVED** [review]
LGTM

**Pull request merged**